datumaro/plugins/data_formats/datumaro_binary/crypter.py

```diff
@@ -0,0 +1,41 @@
+# Copyright (C) 2023 Intel Corporation
```

Comment:
Can we adopt crypter functionality for other formats too in `Environment`?

Reply:
Actually, it will be moved to `datumaro/components` in the next PR https://github.com/openvinotoolkit/datumaro/pull/831 to support media encryption/decryption. We could leave as it is in this PR.